Sources/IO/XML/XMLWriter/index.js

```diff
@@ -0,0 +1,138 @@
+import builder from 'xmlbuilder';
```

Comment:
Best to explicitly add `xmlbuilder` to package.json rather than relying on it being a dependency of xml2js.